docs/templates/themes.html

```diff
@@ -9,6 +9,7 @@
     <div class="themes-container">
         {% block theme_content %}
             <h1>Zola themes</h1>
+            <p><a href="{{ get_url(path = "theme-tags") }}">Search by tags</a></p>
```

Comment:
I think I would wait until we actually have templates with tags before linking to it. No point linking to an empty page.

Reply:
Zluinav theme has that tags. Meanwhile I'll try finding another way out to see if the link can be invisible if there is no defined content with `theme-tags` taxonomy